### ISSUE TITLE
buildDockerImage.sh fault - /opt/oracle/install/installDBBinaries.sh: Permission denied

### ISSUE THREAD

**Issue by juanyque** [oracle/docker-images#321]
buildDockerImage.sh fault on `RUN $INSTALL_DIR/$INSTALL_DB_BINARIES_FILE xx` for all _12_ versions and _ee_ and _se2_ editions (at least):

> Step 8 : USER oracle
 ---> Running in a329d3c92697
 ---> 5761d2f89057
Removing intermediate container a329d3c92697
Step 9 : RUN $INSTALL_DIR/$INSTALL_DB_BINARIES_FILE SE2
 ---> Running in aba118b0bc2a
/bin/bash: /opt/oracle/install/installDBBinaries.sh: Permission denied
Removing intermediate container aba118b0bc2a
The command '/bin/sh -c $INSTALL_DIR/$INSTALL_DB_BINARIES_FILE SE2' returned a non-zero code: 126
There was an error building the image.

I have try to force permissions just before run but it faults again:

> Step 8 : RUN chmod +x $INSTALL_DIR/$INSTALL_DB_BINARIES_FILE
 ---> Running in c63d5ea52182
 ---> b7dec0b99ee6
Removing intermediate container c63d5ea52182
Step 9 : USER oracle
 ---> Running in 8769016e6597
 ---> 7065ef57f02b
Removing intermediate container 8769016e6597
Step 10 : RUN $INSTALL_DIR/$INSTALL_DB_BINARIES_FILE SE2
 ---> Running in 8f12ad91da0c
/bin/bash: /opt/oracle/install/installDBBinaries.sh: Permission denied
Removing intermediate container 8f12ad91da0c
The command '/bin/sh -c $INSTALL_DIR/$INSTALL_DB_BINARIES_FILE SE2' returned a non-zero code: 126
There was an error building the image.

At the end, I have comment the previous `# USER oracle` line in DockerFile and it seems go ahead, but probably with I'll find some issues later by this change... Yes:

> The user is root. Oracle Universal Installer cannot continue installation if the user is root.

**Comment by juanyque** [comment]
It works adding read permissions: `RUN chmod 550 $INSTALL_DIR/$INSTALL_DB_BINARIES_FILE`

**Comment by gvenzl** [comment]
What are the other errors? Are they again about UNIX file permissions?

**Comment by juanyque** [comment]
Hi gvenzl!

Yes, I found some other permissions issues, but after some test I have check that they are related with my environment.

I have a docker environment without git, then I downloaded GitHub project in other server and then move to my docker environment, and this seems to be the issue. It can be solved with some changes on dockerfiles similar to my previous comment, but really installDBBinaries.sh works fine in a full docker/git environment.

Sorry for the inconvenience
